src/ensureCloudResourceLocation.ts

```diff
@@ -0,0 +1,20 @@
+import * as FirebaseError from "./error";
+
+/**
+ * Simple helper function that returns an error with a helpful
+ * message on event of cloud resource location that is not set.
+ * This was made into its own file because this error gets thrown
+ * in several places: in init for Firestore, Storage, and for scheduled
+ * function deployments.
+ * @param location cloud resource location, like "us-central1"
+ * @throws { FirebaseError } if location is not set
+ */
+export function ensureLocationSet(location: string | undefined, feature: string): void {
+  if (!location) {
+    throw new FirebaseError(
+      `Cloud resource location is not set for this project but the operation ` +
```

Comment:
Does our linter actually require line breaks in long strings? It's weird to see backticks and `+`-ing of strings together, I'd probably just have it all as a single line if nothing complains.

Reply:
Just checked - it does complain with `ERROR: /Users/dianatkachenko/code/firebase-tools/src/ensureCloudResourceLocation.ts:15:1 - Exceeds maximum line length of 120`